### PR TITLE
feat: update to ipfs@0.35.0

### PR DESCRIPTION
**Pull request by olizilla**
see: https://github.com/ipfs/js-ipfs/releases/tag/v0.35.0

also adds docs on how you can inspect bundle size with the `vue ui` command.

License: MIT
Signed-off-by: Oli Evans <oli@tableflip.io>

### CODE REVIEW

**Comment by olizilla** [comment]
I was surfacing that `vue ui` gives you a visual tool to discover more about whats going into the bundles. It's part of the `vue` cli tool, and it opens in a browser. I only just found out about it as I was trying to see what improvement updating to the lates js-ipfs would give us in terms of bundle size, but alas, it's not much.

It's not critical, but keeping an eye on the total js bundle size that we're shipping is important. It can be done in CI tho, so I'll remove the docs here in favour of that.

File: README.md [hunk 1]
```diff
@@ -19,11 +19,12 @@ Each **tutorial** in ProtoSchool is made up of multiple **lessons**.
 
 First, check out this repository and run the dev server locally.
 
-```sh
+```console
```
Comment (fsdiogo) [inline README.md]: Why this change? `sh` colors the prompt for me, while `console` doesn't.

**Comment by terichadbourne** [comment]
Thanks for the context, @olizilla. I’ll trust your judgement as to whether it’s helpful to mention the tool, but I suspect we should restructure the headers if we do do it looks like a bonus tip and not like an essential step in authoring a new tutorial.

File: README.md (hunk 1 above)
Comment (olizilla) [reply README.md]: yeah, I did half a change there. `sh` colors `>` because it thinks it is a greater than operator in a shell script. `console` makes more sense for blocks that you want to style as if they were commands entered at a prompt. Refardless, The readme looks better if I leave it as `sh` in this case,  so I'll revert it.

Comment (fsdiogo) [reply README.md]: Hack all the things 👍

**Comment by terichadbourne** [comment]
@olizilla Is this PR where you'd like it now? Are there any differences between 0.34.4 and 0.35.0 that would justify working through all the individual lessons to ensure they're still working?

**Comment by terichadbourne** [comment]
@olizilla Is this PR where you'd like it now? Are there any differences between 0.34.4 and 0.35.0 that would justify working through all the individual lessons to ensure they're still working? I’m not sure how best to test this one.

**Comment by terichadbourne** [comment]
@olizilla I made a quick attempt to proof this by merging the code branch into it locally (to enable the View Solution feature), but hit merge conflicts in the package-lock.json file. Normally I can handle merging but I screwed it up royally the last time I did it on that particular file so I'm wary. Happy to look at it together tomorrow manually or wait for you to finish building your testing robot if this isn't urgent and you're still having fun with that. Just let me know what sounds good.

**Comment by olizilla** [comment]
yerp, package-lock.json needs extra magic. I use this https://www.npmjs.com/package/npm-merge-driver - we can take a look tomorrow

**Comment by fsdiogo** [comment]
Merged this one using npm-merge-driver, everything seem OK.

**Comment by fsdiogo** [comment]
@olizilla @terichadbourne can this be merged?

**Comment by terichadbourne** [comment]
Everything's working for me as well. I'll go ahead and merge and we can do the robot testing on the code branch.
